Customer Group and Shared Catalog limitations by Websites (#1087)
magento/merchdocs: src/customers/account-scope.md

   If necessary, clear the **User system value** checkbox to make the change.

1. When complete, click <span class="btn">Save Config</span>.

> {:.bs-callout-info}

Reviewer: ```suggestion
   {:.bs-callout-info}
```

> If [website is excluded from the customer group](https://devdocs.magento.com/guides/v2.4/extension-dev-guide/indexer-optimization.html#customer-group-limitations-by-websites), customer is not allowed to login to the website if the scope of customer accounts is limited to the website or shared will all websites.

Reviewer: ```suggestion
If the [website is excluded from the customer group](https://devdocs.magento.com/guides/v2.4/extension-dev-guide/indexer-optimization.html#customer-group-limitations-by-websites), the customer is not allowed to log in to the website when the scope of customer accounts is limited to the website or shared will all websites.
```

> If the [website is excluded from the customer group](https://devdocs.magento.com/guides/v2.4/extension-dev-guide/indexer-optimization.html#customer-group-limitations-by-websites), the customer is not allowed to log in to the website when the scope of customer accounts is limited to the website or shared with all websites.

Reviewer: ```suggestion
   If the [website is excluded from the customer group](https://devdocs.magento.com/guides/v2.4/extension-dev-guide/indexer-optimization.html#customer-group-limitations-by-websites), the customer is not allowed to log in to the website when the scope of customer accounts is limited to the website or shared with all websites.
```
